networkx/generators/duplication.py

```diff
@@ -68,7 +72,7 @@ def partial_duplication_graph(N, n, p, q, seed=None):
     if n > N:
         raise NetworkXError("partial duplication graph must have n <= N.")
 
-    G = nx.complete_graph(n)
+    G = nx.complete_graph(n, create_using)
```

Comment:
To be safe, I think we would need to add an additional check to disallow directed (definitely) and multigraphs (maybe) in order to maintain consistency with the original algorithm.